postman/threads-api.postman_collection.json

```diff
@@ -2218,6 +2519,269 @@
+									"description": "This is the query to be searched for."
+								},
+								{
+									"key": "search_type",
+									"value": "TOP",
+									"description": "This is the type of search to be performed. This can be either TOP or RECENT."
+								},
+								{
+									"key": "limit",
+									"value": "",
+									"description": "Query string parameter representing the maximum number of media objects or records to return.",
+									"disabled": true
+								},
+								{
+									"key": "fields",
+									"value": "{{fields_threads}}",
+									"description": "A comma-separated list of fields for media objects on Threads."
+								}
+							]
+						}
+					},
+					"response": [
+						{
+							"name": "Search for Threads Posts",
+							"originalRequest": {
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{api_host}}/keyword_search?q=ThreadsAPI&search_type=TOP&fields={{fields_threads}}",
+									"host": [
+										"{{api_host}}"
+									],
+									"path": [
+										"keyword_search"
+									],
+									"query": [
+										{
+											"key": "q",
+											"value": "ThreadsAPI",
+											"description": "This is the query to be searched for."
+										},
+										{
+											"key": "search_type",
+											"value": "TOP",
+											"description": "This is the type of search to be performed. This can be either TOP or RECENT."
+										},
+										{
+											"key": "limit",
+											"value": "",
+											"description": "Query string parameter representing the maximum number of media objects or records to return.",
+											"disabled": true
+										},
+										{
+											"key": "fields",
+											"value": "{{fields_threads}}",
+											"description": "A comma-separated list of fields for media objects on Threads."
+										}
+									]
+								}
+							},
+							"_postman_previewlanguage": null,
+							"header": null,
+							"cookie": [],
+							"body": "{\n    \"data\": [\n        {\n            \"id\": <string>,\n            \"media_product_type\": \"THREADS\",\n            \"media_type\": \"TEXT_POST\",\n            \"permalink\": <string>,\n            \"username\": <string>,\n            \"text\": <string>,\n            \"timestamp\": \"2025-01-01T00:00:00+0000\",\n            \"shortcode\": <string>,\n            \"is_quote_post\": false,\n            \"has_replies\": false\n        },\n        ...\n    ],\n    \"paging\": {\n        \"cursors\": {\n            \"before\": <string>,\n            \"after\": <string>\n        }\n    }\n}"
+						}
+					]
+				},
+				{
+					"name": "Get Mentioned Threads Posts",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{api_host}}/me/mentions?fields={{fields_threads}}",
+							"host": [
+								"{{api_host}}"
+							],
+							"path": [
+								"me",
+								"mentions"
+							],
+							"query": [
```

Comment:
Should we add the `before` and `after` parameters to the mentions endpoint?

https://developers.facebook.com/docs/threads/reference/user#get---threads-user-id--mentions

Reply:
Done!